src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -157,6 +158,15 @@ public void load(CoreExtension.Context context) {
                                           .build(),
                 MonoRepoFeature.class);
 
+            context.addExtensions(PropertyDefinition.builder(PR_SUMMARY_NOTE_FIRST)
+                    .category(getName())
+                    .subCategory("Merge Request Decoration")
+                    .onQualifiers(Qualifiers.PROJECT)
+                    .name("Submit summary note first")
+                    .description("Submit summary discussion thread before issue comments.")
+                    .type(PropertyType.BOOLEAN)
+                    .defaultValue(String.valueOf(false))
+                    .build());
```

Comment:
I'm not sure this needs a configuration property. I'm happy that the plugin should remain opinionated and only implement configuration options that Sonarqube's core provides. I'd remove this and just go with the Summary note always being first if that's the direction you think the plugin should be going in